fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import UserList from './user-list';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      isLoading: true,
+      isError: false
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://jsonplaceholder.typicode.com/users', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify()
+    })
+      .then(res => {
+        return res.json();
+      })
+      .then(this.setState({ isLoading: false }))
```

Comment:
This function will run right away, it needs to be wrapped in an anonymous function like the other two callbacks below. Also you can set this when you set the user data. Example below